js/stdlib/ts/ALGO.ts

```diff
@@ -1451,13 +1451,31 @@ export const connectAccount = async (networkAccount: NetworkAccount): Promise<Ac
       };
     };
 
+
+    const stateCache = new Map<GetC, { time: number, val: [AppStateKVs, GlobalState] }>();
```

Comment:
This is really weird now... now you're making it so it is indexed on the `GetC` value which is a lambda for the whole contract representation. I was just thinking that there'd be one box AS PART OF `getC` (another field) that was the cache, or rather that internal to `getC`... `getAppState` and `getGlobalState` were themselves caching